lesson_27/kimberleeh/src/App.tsx

```diff
@@ -0,0 +1,38 @@
+import './App.scss';
+
+import { Link, Outlet } from 'react-router-dom';
+
+
+
+function App() {
+  return (
+    <>
+      <nav className="navbar">
```

Comment:
Why remove the `Header` and `Footer` components from here? You should have added your links to the Header component that's already been setup and styled for you.